nxdl.xsd

```diff
@@ -502,7 +502,8 @@ https://stackoverflow.com/a/48980995/1046449 -->
 		<xs:attribute name="optional" use="optional" type="nx:NX_BOOLEAN" default="false" >
 			<xs:annotation>
 				<xs:documentation>
-					A synonym for minOccurs=0.
+					The value "true" is equivalent to minOccurs=0 while the value
+					"false" is equivalent to minOccurs>0.
```

Comment:
Are the choices as large as `minOccurs>0`?  Or the only choice when `false` is `minOccurs=1`?

Reply:
> Are the choices as large as minOccurs>0?

Any nonNegativeUnbounded greater than zero.